### ISSUE TITLE
useLaravelPint

### ISSUE THREAD

**Issue by OlyaK95** [fortephp/chisel#117]
I have three files:
- .prettierrc

```json
{
    "plugins": ["@prettier/plugin-php", "prettier-plugin-blade", "prettier-plugin-tailwindcss"],
    "overrides": [
        {
            "files": ["*.php"],
            "options": {
                "parser": "php"
            }
        },
        {
            "files": ["*.blade.php"],
            "options": {
                "parser": "blade"
            }
        }
    ]
}
```
- .blade.format.json
```json
{
    "useLaravelPint": true
}
```
- pint.json
```json
{
    "preset": "psr12"
}
```
When I run command like `bunx prettier resources/ --write` I have no Laravel Pint ouput.
Why?

**Comment by JohnathonKoster** [comment]
>  I have no Laravel Pint ouput.

Do you see any files formatted/etc. at all? The pint output generally is not output directly with the way it has been integrated, as its only applied inside directives and PHP blocks within Blade files.

**Comment by OlyaK95** [comment]
> > I have no Laravel Pint ouput.
> 
> Do you see any files formatted/etc. at all? The pint output generally is not output directly with the way it has been integrated, as its only applied inside directives and PHP blocks within Blade files.

Yes, I see that Prettier working, but I don't know this about Laravel Pint.
![2024-09-27_18-04-09](https://github.com/user-attachments/assets/4235c9ed-45ed-4973-8a54-1e76f16870ee)

**Comment by OlyaK95** [comment]
> > I have no Laravel Pint ouput.
> 
> Do you see any files formatted/etc. at all? The pint output generally is not output directly with the way it has been integrated, as its only applied inside directives and PHP blocks within Blade files.

I set `laravel` preset in `pint.json` and do some test on `routes/web.php` file.
When I run only Pint I have singe quotes.
When I run Prettier I have double qoutes.

**Comment by JohnathonKoster** [comment]
Hm interesting. I'll need to experiment and see what could be going on. Thanks for providing the config files!

**Comment by OlyaK95** [comment]
> Hm interesting. I'll need to experiment and see what could be going on. Thanks for providing the config files!

Maybe issue about Windows OS? For example, I run Pint with the command `"./vendor/bin/pint"`, not the `./vendor/bin/pint`. Only my guess.

**Comment by JohnathonKoster** [comment]
I am going to close pint related issues for now to help me maintain/manage the project as I prep a v3 release.
